### DISCUSSION TITLE
Not syching on Rasberry with Ubuntu

### DISCUSSION THREAD

**Discussion by chiachain** [Chia-Network/chia-blockchain#1829] [Q&A]
With a fresh install of 1.0.3 on Ubuntu using the terminal commands, the UI is up and running.  I imported my keys, and am connected, but..... only connected to Farmer and Wallet - no Full Nodes, and not syching.

Any suggestions?  Thanks!!!

**Comment by dorkmo** [comment]
i just installed ubuntu server on my pi 400, had to use the `chia show -a ip:port` command to give it some known peers

it can take a long time to get peers on its own
